quantum/quantum.h

```diff
@@ -16,32 +16,33 @@
 #ifndef QUANTUM_H
 #define QUANTUM_H
 
-#if defined(__AVR__)
+#ifdef __AVR__
     #include <avr/pgmspace.h>
     #include <avr/io.h>
     #include <avr/interrupt.h>
 #endif
-#if defined(PROTOCOL_CHIBIOS)
+#ifdef PROTOCOL_CHIBIOS
     #include "hal.h"
 #endif
+
 #include "wait.h"
 #include "matrix.h"
 #include "keymap.h"
+
 #ifdef BACKLIGHT_ENABLE
     #ifdef LED_MATRIX_ENABLE
         #include "ledmatrix.h"
     #else
         #include "backlight.h"
     #endif
 #endif
-#ifdef RGBLIGHT_ENABLE
-  #include "rgblight.h"
-#else
-    #ifdef RGB_MATRIX_ENABLE
-        /* dummy define RGBLIGHT_MODE_xxxx */
-        #define RGBLIGHT_H_DUMMY_DEFINE
-        #include "rgblight.h"
-    #endif
+
+#if defined(RGBLIGHT_ENABLE)
+    #include "rgblight.h"
+#elif defined(RGB_MATRIX_ENABLE)
```

Comment:
#elseif can still use the short form, I think if you are going to standardize, it should probably be: short form for single defines, long form otherwise.

Reply:
Could you elaborate a bit on this? There isn't an `#elifdef` directive as far as I'm aware, and `#elif X` isn't exactly equivalent to `#elif defined(X)`. This is why I opted to go with `#elif defined(X)` with and a matching `#if defined(X)` here.

Reply:
err, ya, you're right, my fault.